packages/quick_start/source/README.md

```diff
@@ -30,10 +30,10 @@ You can use it as a baseline to write your own first kusto client application, a
 ### Run the app
 
 1. Open a command line window and navigate to the folder where you extracted the app.
-2. Either use an IDE of choice to build and run the project, or do the following using the command line window:
-   1. Run `npm install` in the folder
-   2. Run `npm run quick_start`
+2. Either use an IDE of your choice, or do the following using the command line window:

```

Comment:
By my comment I meant to fix the typos/phrasing that I quoted, so you don't need to erase, and I do prefer "execute the following" over "do the following"